turtlebot3_automatic_parking/setup.py

```diff
@@ -22,8 +18,8 @@
     ],
     install_requires=['setuptools', 'launch'],
     zip_safe=True,
-    author=['Gilbert'],
-    author_email=['kkjong@robotis.com'],
+    author=['Gilbert, YeonSoo Noh'],
+    author_email=['kkjong@robotis.com, nys8277@gmail.com'],
```

Comment:
```suggestion
    author=['Gilbert', 'YeonSoo Noh'],
    author_email=['kkjong@robotis.com', 'nys8277@gmail.com'],
```
Please revise it as follows.

Reply:
[153ade7](https://github.com/ROBOTIS-GIT/turtlebot3_applications/pull/50/commits/153ade7de7a205b8b1e0be893dead51e404d2604)
Fixed.